pkg/ttl/cache/task.go

```diff
@@ -122,9 +122,9 @@ type TTLTaskState struct {
 }
 
 // RowToTTLTask converts a row into TTL task
-func RowToTTLTask(sctx sessionctx.Context, row chunk.Row) (*TTLTask, error) {
+func RowToTTLTask(se session.Session, row chunk.Row) (*TTLTask, error) {
```

Comment:
How about also use `loc *time.Location` here? `se` is not used in this function 🤔 .